mkosi/archive.py

```diff
@@ -89,6 +89,8 @@ def extract_tar(src: Path, dst: Path, log: bool = True) -> None:
 def make_cpio(src: Path, dst: Path, files: Optional[Iterable[Path]] = None) -> None:
     if not files:
         files = src.rglob("*")
+    files = list(files)
+    files.sort()
```

Comment:
```suggestion
        files = sorted(src.rglob("*"))
```

Reply:
That if the caller passed `files` as an unsorted iterable?

Reply:
Then 
```suggestion
        files = src.rglob("*")
    files = sorted(files)
```